tests/unit/amo/helpers.js

```diff
@@ -137,3 +137,14 @@ export function createAddonsApiResult(results) {
   // For example: core.api.featured(), core.api.search()...
   return normalize({ results }, { results: [addonSchema] });
 }
+
+export function createFakeAutocompleteResult(name, lang = 'en-US') {
```

Comment:
again, here, I'd like to functions using a single object for multiple parameters, like `createFakeAutocompleteResult({ name, lang = 'en-US' })`